src/formatters/sr_formatter.erl

```diff
@@ -0,0 +1,54 @@
+%%% @doc Formatter to integrate with
+%%%      <a target="_blank" href="https://github.com/old-reliable/steamroller">steamroller</a>.
+-module(sr_formatter).
+
+-behaviour(rebar3_formatter).
+
+-export([format/2]).
+
+%% @doc Format a file.
+%%      Apply formatting rules to a file containing erlang code.
+%%      Use <code>Opts</code> to configure the formatter.
+-spec format(file:filename_all(), rebar3_formatter:opts()) -> rebar3_formatter:result().
+format(File, OptionsMap) ->
+    Opts = maps:fold(fun parse_opt/3, [], OptionsMap),
+    FileToFormat = case maps:get(output_dir, OptionsMap, current) of
+                     current ->
+                         File;
+                     none ->
+                         File;
+                     OutputDir ->
+                         copy_file(File, OutputDir)
+                   end,
+    {ok, Code} = file:read_file(FileToFormat),
+    case steamroller_formatter:format(iolist_to_binary(FileToFormat), Opts) of
```

Comment:
```suggestion
    case steamroller:format_file(iolist_to_binary(FileToFormat), Opts) of
```
Better to use the main steamroller API here. The steamroller_formatter API isn't likely to change but it might!